lib/cloud_controller/diego/docker/staging_completion_handler.rb

```diff
@@ -12,8 +12,9 @@ def initialize(runners)
         def self.success_parser
           @staging_response_schema ||= Membrane::SchemaParser.parse do
             {
-              execution_metadata: String,
-              detected_start_command: Hash,
+              :execution_metadata => String,
```

Comment:
We re-use the lifecycle_metadata to return the cached/generated docker image. This field however is optional (users can opt-in for caching). Another reason for having it optional is that until we merge the changes in Diego the response won't contain the cached image anyway.

The new syntax is fine until we have `optional(:symbol)`. Seems this requires the old syntax. We saw the same pattern in all places where `optional` is used. We can use the new hash syntax for the first 2 fields and the old for the optional `lifecycle_data`, but this does not improve the block in my opinion.